src/cbmc/cbmc_parse_options.cpp

```diff
@@ -926,6 +953,8 @@ void cbmc_parse_optionst::help()
     " --error-label label          check that label is unreachable\n"
     " --cover CC                   create test-suite with coverage criterion CC\n" // NOLINT(*)
     " --mm MM                      memory consistency model for concurrent programs\n" // NOLINT(*)
+    HELP_REACHABILITY_SLICER
+    " --full-slice                 run full slicer (currently buggy)\n" // NOLINT(*)
```

Comment:
I think the "currently buggy" is not very helpful to users - when exactly is it broken? If you insist on marking it in some way, I'd use "experimental".

Reply:
Fair point. Changed to "experimental". The reason I added this line at all was that the option is accepted already (before this PR), and I felt the need to sync the help message with the actual list of switches.